### PR TITLE
Users screen fetch update

### PR DESCRIPTION
**Pull request by dvira**
- Added 'Load More' button and utilize it (fetch 10 more users on each click). Also added users count indication.
- Integrated users search filter with SDK.
- Added debounce hook for search filter.
- Updated `SearchFilter` skeleton and styles.
- Updated `ShownUsersIndicator`: Fixed behavior and added skeleton.
- Updated empty users table styles.
- Tweaked courses screen 'create' button to have fixed width.

### CODE REVIEW

File: src/components/Screen/UsersScreen/ShownUsersIndicator.tsx [hunk 1]
```diff
@@ -2,21 +2,31 @@ import React, { ReactElement } from 'react';
...
+      className={classes['shown-users-skeleton']}
+      loading={isFetchingUsers}
+      active
+      paragraph={false}
+    >
+      <div className={classes['shown-users-indicator']}>
+        {usersSearchValue ? (
+          <>
+            Search
+            <span className={classes['search-result']}>{total_rows} Results found</span>
```
Comment (dr15) [inline src/components/Screen/UsersScreen/ShownUsersIndicator.tsx:24]: Is it possible to have just one result? How about adding the "pluralizer" you added in a previous PR?

Comment (dr15) [inline src/components/Screen/UsersScreen/ShownUsersIndicator.tsx:24]: Is it possible to have just one result? How about adding the "pluralizer" you added in a previous PR?

File: src/components/Screen/UsersScreen/ShownUsersIndicator.tsx [hunk 2]
```diff
@@ -2,21 +2,31 @@ import React, { ReactElement } from 'react';
...
+          </>
+        ) : (
+          <>{totals_unique_users} Users</>
```
Comment (dr15) [inline src/components/Screen/UsersScreen/ShownUsersIndicator.tsx:27]: This also seems like a good place for the "pluralizer"

File: src/components/Screen/UsersScreen/ShownUsersIndicator.tsx (hunk 1 above)
Comment (dvira) [reply src/components/Screen/UsersScreen/ShownUsersIndicator.tsx]: Done

File: src/components/Screen/UsersScreen/ShownUsersIndicator.tsx (hunk 2 above)
Comment (dvira) [reply src/components/Screen/UsersScreen/ShownUsersIndicator.tsx]: Done

**Pull request merged**